apps/nova/src/components/Settings/ProfileForm.tsx

```diff
@@ -217,6 +220,8 @@ export const ProfileForm: React.FC<{ user: IUser.Self }> = ({ user }) => {
       <div className=" flex flex-col gap-6">
         <UploadPhoto
           id={user.id}
+          // Teachers can't change their photo on their own
+          isAllowed={user.role === IUser.Role.Student}
```

Comment:
No need to have this check. Tutors will have their own settings page. We can safely remove this.